Detectors/TPC/qc/macro/runPID.C

```diff
@@ -0,0 +1,61 @@
+using namespace o2::tpc;
```

Comment:
done

Reply:
On my local machine now everything compiles including the macros. I can also compile on the fly when calling the macro with root. 
Here the checks fail because TPCQC/Clusters.h and TPCQC/PID.h are not found in the corresponding macro. Any idea why?

Reply:
Hmm, funny. I think this should work. We can check in separately.